### PR TITLE
update radioSchedule config with placeholders for header and frequencies link text

### LINKED ISSUE #5810: Add header and frequencies label placeholders to Radio Schedule config
**Is your feature request related to a problem? Please describe.**
Add `header: 'BBC News Radio'` and `frequenciesPageLabel: 'Radio Frequencies Link Label'` to `radioSchedule` in the config for respective services

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by itimirichard**
Resolves #5810

**Overall change:** 
_Updated the respective radio schedule services with `header` and `frequenciesPageLabel` placeholders
```
frequenciesPageLabel: 'Radio Frequencies Link Label'
header: 'BBC News Radio'
```
._

**Code changes:**
- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by Bopchy — APPROVED** [review]

File: src/app/lib/config/services/korean.js [hunk 1]
```diff
@@ -154,6 +154,7 @@ export const service = {
     },
     radioSchedule: {
       hasRadioSchedule: true,
+      header: 'BBC News Radio',
```
Comment (OlgaLyubin) [inline src/app/lib/config/services/korean.js:157]: Why do some services have `frequenciesPageLabel` and some don't?

**Review by EinsteinNjoroge — COMMENTED** [review]
🤔 Shouldn't we have localised `headers` and `pageLabels` for the different services?

File: src/app/lib/config/services/korean.js (hunk 1 above)
Comment (itimirichard) [reply src/app/lib/config/services/korean.js:157]: @OlgaLyubin Only services that have `frequenciesPageUrl` should have a corresponding `frequenciesPageLabel`

**Comment by itimirichard** [comment]
@EinsteinNjoroge  We definitely would have localised headers and pageLabels. These are placeholders until we have these translations

**Review by OlgaLyubin — APPROVED** [review]
LGTM

**Review by EinsteinNjoroge — APPROVED** [review]

**Pull request merged**